content/blog/en/version-7-dojo.md

```diff
@@ -0,0 +1,329 @@
+    const [ child ] = children();
+    const active = icache.getOrSet('active', true);
+    return (
+        <div>{child(active)}</div>
+        <button onclick={() => {
+            icache.set('active', icache.getOrSet('active', true));
+        }}>{`Set ${active ? 'inactive' : 'active'}`}</button>
+    );
+});
+
+// Usage
+<MyWidget>{(active) => <div>{`${active ? 'ACTIVE' : 'NOT ACTIVE'}`}</div>}</MyWidget>
+```
+
+Typed children can be even more expressive by using an object that describes children for different sections of the widget, referred to as named children. For example, a `Card` widget could have `title`, `avatar`, and `content` that can get defined by a user. Using standard children, there is no clear way for a user to define the output required for each, and using render properties still have all the original downsides of the pattern. However with named children, an object can be used to indicate the purpose of the child and whether they are mandatory or optional.
+
+```tsx
+import { create, tsx } from '@dojo/framework/core/vdom';
+
+import * as css from './Card.m.css';
+
+interface CardChildren {
+    title: RenderResult;
+    avatar: RenderResult;
+    content: RenderResult;
+}
+
+const factory = create().children<CardChildren>();
+
+const Card = factory(function Card({ children }) {
+    const [{ title, avatar, content }] = children();
+    return (
+        <div>
+            <div classes={[css.title]}>
+                {title}
+            </div>
+            <span classes={[css.avatar]}>
+                {avatar}
+            </span>
+            <div classes={[css.content]}
+                {content}
+            </div>
+        </div>
+    );
+});
+
+// Usage
+<Card>{{
+    title: 'My Title',
+    avatar: <img src="https://my-content.com/avatar.png"/>,
+    content: <div>My Main Content!</div>
+}}</Card>
+```
+
+## Custom Elements, Improved
+
+Dojo's support for the Custom Elements portion of Web Components provides interoperability with other frameworks and component systems, not only consuming custom elements in the framework but building Dojo widgets as custom elements. Developer ergonomics are even more important with custom elements, to ensure that the Dojo custom elements can get used with minimal effort, in the declarative way they get designed.
+
+Working with Dojo 6 compiled custom elements that used the render property pattern did not allow the custom elements to be used effectively, however, in Dojo 7 widgets that use functional children in place of a render property can use the custom element declaratively like standard HTML.
+
+```html
+<dojo-my-widget>
+	<div>My content for the widget</div>
+</dojo-my-widget>
+```
+
+For widgets that take advantage of the named children pattern, the children can also be declaratively defined using the new "slots" attribute to indicate which section of the widget's children the node is intended for. Using the `Card` widget example above, this would look something like:
```

Comment:
I feel like it's important to note whether this maps to the Custom Element slot concept or if it is different.

Reply:
@dylans I added something below